.github/workflows/packaging.yml

```diff
@@ -37,7 +37,7 @@ jobs:
       DIST: ${{ matrix.platform.dist }}
 
     steps:
-      - name: Clone the module
+      - name: Clone the connector
```

Comment:
Not related with the patch.
https://github.com/tarantool/tarantool/wiki/Code-review-procedure#general-coding-points-to-check
`Don't do a change if it is not related to the patch...`

Reply:
I discussed this. Now this change is added to the second commit.